js/presentation.js

```diff
@@ -4,6 +4,11 @@ import Slide from "components/Slide.react";
 import Code from "components/Code.react";
 import DocumentTitle from "react-document-title";
 
+// Prism JS configuration
+import 'prismjs/components/prism-json';
+import 'prismjs/components/prism-jsx';
+import "style!css!prismjs/themes/prism.css"; // Theme
```

Comment:
Would be great, I think. But it was crucial for me to have it done for the yesterday's presentation. So it could be a next step of moving it to the config.